challenges/objects-arrays.js

```diff
@@ -1,125 +1,191 @@
 
-
 // ==== ADVANCED Array Methods ====
 
 // Given this zoo data from around the United States, follow the instructions below.  Use the specific array methods in the requests below to solve the problems.
 
-
-zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":"Canis aureus","state":"Kentucky"},
-{"animal_name":"Screamer, southern","population":1,"scientific_name":"Chauna torquata","state":"Alabama"},
-{"animal_name":"White spoonbill","population":8,"scientific_name":"Platalea leucordia","state":"Georgia"},
-{"animal_name":"White-cheeked pintail","population":1,"scientific_name":"Anas bahamensis","state":"Oregon"},
-{"animal_name":"Black-backed jackal","population":2,"scientific_name":"Canis mesomelas","state":"Washington"},
-{"animal_name":"Brolga crane","population":9,"scientific_name":"Grus rubicundus","state":"New Mexico"},
-{"animal_name":"Common melba finch","population":5,"scientific_name":"Pytilia melba","state":"Pennsylvania"},
-{"animal_name":"Pampa gray fox","population":10,"scientific_name":"Pseudalopex gymnocercus","state":"Connecticut"},
-{"animal_name":"Hawk-eagle, crowned","population":10,"scientific_name":"Spizaetus coronatus","state":"Florida"},
-{"animal_name":"Australian pelican","population":5,"scientific_name":"Pelecanus conspicillatus","state":"West Virginia"}];
+zooAnimals = [
+  { animal_name: 'Jackal, asiatic', population: 5, scientific_name: 'Canis aureus', state: 'Kentucky' },
+  { animal_name: 'Screamer, southern', population: 1, scientific_name: 'Chauna torquata', state: 'Alabama' },
+  { animal_name: 'White spoonbill', population: 8, scientific_name: 'Platalea leucordia', state: 'Georgia' },
+  { animal_name: 'White-cheeked pintail', population: 1, scientific_name: 'Anas bahamensis', state: 'Oregon' },
+  { animal_name: 'Black-backed jackal', population: 2, scientific_name: 'Canis mesomelas', state: 'Washington' },
+  { animal_name: 'Brolga crane', population: 9, scientific_name: 'Grus rubicundus', state: 'New Mexico' },
+  { animal_name: 'Common melba finch', population: 5, scientific_name: 'Pytilia melba', state: 'Pennsylvania' },
+  { animal_name: 'Pampa gray fox', population: 10, scientific_name: 'Pseudalopex gymnocercus', state: 'Connecticut' },
+  { animal_name: 'Hawk-eagle, crowned', population: 10, scientific_name: 'Spizaetus coronatus', state: 'Florida' },
+  {
+    animal_name: 'Australian pelican',
+    population: 5,
+    scientific_name: 'Pelecanus conspicillatus',
+    state: 'West Virginia',
+  },
+];
 
 /* Request 1: .forEach()
 
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
 const animalNames = [];
+zooAnimals.forEach(animal => {
+  animalNames.push({
+    Name: animal.animal_name,
+    Scientific: animal.scientific_name,
+  });
```

Comment:
I like your thought process here, I think we're looking for a string value, though:
```
animalNames.push(`Name: ${animal.animal_name}, Scientific: ${animal.scientific_name}`))
```